back/identify/accounts/models.py

```diff
@@ -0,0 +1,33 @@
+import imp
+from django.db import models
+from django.contrib.auth.models import User
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from django.utils import timezone
+from django.utils.translation import gettext_lazy as _
+from Google.models import Google
+from LinkedIn.models import LinkedIn
+from Github.models import Github
+
+
+# Create your models here.
+
+
```

Comment:
set on_delete to models.SET_NULL 
we don't want to delete the whole profile just because the object in one of the fields is deleted